Gopkg.lock

```diff
@@ -73,12 +73,12 @@
   version = "v23.2.0"
 
 [[projects]]
-  digest = "1:b15d5bdadce5d98f1e06353508a4029fccfeb68761957b3a2a8cb38ebb8caca4"
+  digest = "1:9d3a690fcd29a7efbc783d7a8c9c173c2c67d33c0d2b6ac4be10c78d4ae46df2"
   name = "github.com/Azure/azure-storage-blob-go"
   packages = ["azblob"]
   pruneopts = "UT"
-  revision = "678206e7e6e55abf0265a6440135e92005176ebf"
-  version = "v0.6.0"
```

Comment:
These changes are not correct.
please update this file with:
dep ensure --update github.com/Microsoft/presidio-genproto

Reply:
Fixed (by you...)